src/pages/chat/components/UploadForm/const.ts

```diff
@@ -21,8 +21,11 @@ export const validationOfFile = ({
   supportedFormats?: string[];
 }) => {
   const formats = supportedFormats || SUPPORTED_FILE_FORMATS;
-  return Yup.mixed()
-    .nullable(isOptional ? true : undefined)
+  let schema = Yup.mixed();
+  if (!isOptional) {
```

Comment:
isnt this the other way?
```suggestion
  if (isOptional) {
```

Reply:
good catch